mandatory/3-function-output.js

```diff
@@ -11,6 +11,7 @@ function combine2Words(word1, word2) {
 function concatenate(firstWord, secondWord, thirdWord) {
```

Comment:
Can you please read the comments above what this exercise is asking you to do (i.e. add comments to explain what the function does, which is missing from this)